ksqldb-rest-app/src/test/java/io/confluent/ksql/api/utils/PrintResponse.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.utils;
+
+import io.vertx.core.json.Json;
+import io.vertx.core.json.JsonObject;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+
+public class PrintResponse {
+
+  public final List<String> rows;
+  public final JsonObject error;
```

Comment:
would be better to make this private with getters (and don't provide setters if not needed)